tap_hubspot/__init__.py

```diff
@@ -499,7 +589,7 @@ def sync_deals(STATE, ctx):
     max_bk_value = start
     LOGGER.info("sync_deals from %s", start)
     most_recent_modified_time = start
-    params = {'count': 250,
+    params = {'limit': 100,
```

Comment:
I didn't ask last time, but is there a reason we dropped this to paginate by 100 and not by 250?

Reply:
The V3 endpoint has a limit of 100. So I needed the V1 endpoint to match it in order for my "search for this deal IDs" strategy to work without pagination